MainModule/Server/Core/Functions.lua

```diff
@@ -1114,7 +1115,7 @@ return function(Vargs, GetEnv)
 
 		Shutdown = function(reason)
 			Functions.Message("Server Shutdown", "The server is shutting down...", service.Players:GetPlayers(), false, 5)
-			wait(1)
+			task.wait(1)
```

Comment:
Doesn't `wait()` already resolve to `task.wait()` at the top of the file? I'm sure functions already has `wait = task.wait` at the top.